crates/containerd-shim-wasmtime/src/instance.rs

```diff
@@ -372,6 +371,8 @@ mod wasitest {
         let output = read_to_string(dir.path().join("stdout"))?;
         assert_eq!(output, "hello world\n");
 
+        wasi.delete()?;
```

Comment:
Would it make sense to have as part of `trait Drop`?

Reply:
I'm not sure! It would look nicer but we would lose the error handling if `delete()` fails. Also it would be a breaking change in the API. I would maybe not add that change into this PR but think about it later.